### DISCUSSION TITLE
Receiving CORS errors

### DISCUSSION THREAD

**Discussion by standingdreams** [magicbell/community#195] [Questions]
I am implementing Magic Bell in a Next.js project with the pages router. I'm using the React SDK, and things seemed to be going well. I was able to receive push notifications in Chrome, Firefox, and Edge. However, when I tested on Safari, I didn't receive a push notification. Then I moved to my mobile device, and not only was I not getting push notifications, but I also was not receiving real-time notifications. I then noticed I was not receiving real-time notifications in any browser. I have to refresh the page to see real-time notifications.

That's when I opened up my devtools console. I'm seeing the same CORS errors from [this discussion](https://github.com/orgs/magicbell/discussions/179) that was closed. It happens while working locally, whether using localhost or HTTPS (via Ngrok).

<img width="730" alt="image" src="https://github.com/magicbell/community/assets/942959/e7b12e97-b940-4232-98cc-f0d4ac9caeec">

While attempting to answer the questions below, I opened the site up in incognito and ran across these errors: 

<img width="741" alt="image" src="https://github.com/magicbell/community/assets/942959/5c84139f-befa-4f52-9972-81f41d17a9b0">

- Is this on a corporate/public network, or at home? - **at home**
- any chance that a proxy is removing headers? - **No**
- do you have the same issue on another device? - **Yes**
- if you do have the ability to test on another network using the same device, does the issue occur on another network as well? - **Tested on another network from the same device and get the same issue.**

Any help would be greatly appreciated. Thank you.

**Comment by unamashana** [comment]
Hey @standingdreams I am sorry that we are having this issue again. We are in the process of reverting back to websockets (from SSE), and also dropping Ably in favor of an inhouse solution (though that'd take us some time). We are aware of the /inbox/installations issue (it's annoying but harmless), but the /ws/auth failure shouldn't be happening. Are you able to inspect the headers on that call to check that the correct magicbell auth headers are provided? Do you get a response body back on it?

**Reply by standingdreams** [reply]
@unamashana I'm unable to replicate the `ws/auth` failures anymore. I'll keep trying and will report back if I am finally able to.

In the meantime, while you work on this, should I downgrade to a version before moving to Ably? It seems that in the previous thread, someone downgraded to version 4.0.3 to get things to work properly.

**Reply by standingdreams** [reply]
Note: tried to downgrade. No luck.
